codalab/lib/bundle_cli.py

```diff
@@ -1375,7 +1375,7 @@ def do_upload_command(self, args):
                     'Attempted to upload bundle of size %d with only %d remaining in user\'s disk quota.'
                     % (total_bundle_size, disk_left)
                 )
-
+            print("Bundle of size %d meets the disk quota requirement of %d, continuing..." % (total_bundle_size, disk_left), file=self.stderr)
```

Comment:
Could we print out a message earlier 'Checking files...{size} so far...\r' to have a bit more of a status message, and then make sure at the end of the day, only the uuid shows up in the stdout.

Reply:
sorry, why would we print `Checking files...{size}`? The user gets the information regardless, either they get the `DiskQuotaExceededError` or the print statement after. And everything goes to `stderr` here.